### PR TITLE
CA-561: Building Airflow, pushing to ECR

### PR DESCRIPTION
**Pull request by nkijak**

### CODE REVIEW

File: docker-compose-LocalExecutor.yml [hunk 1]
```diff
@@ -13,8 +13,9 @@ services:
         depends_on:
             - postgres
         environment:
-            - LOAD_EX=y
+            - LOAD_EX=n
             - EXECUTOR=Local
+            - FERNET_KEY=U_HRDNLt20TjStfKkTlopJW7wGcJbExlmknp-m4zzMg=
```
Comment (stevelevine) [inline docker-compose-LocalExecutor.yml:18]: Wat?

File: script/entrypoint.sh [hunk 2]
```diff
@@ -7,11 +7,11 @@ TRY_LOOP="20"
...
 
-: ${POSTGRES_HOST:="postgres"}
+: ${POSTGRES_HOST:="cde-airflow.c1a1plgrhwop.us-east-1.rds.amazonaws.com"}
 : ${POSTGRES_PORT:="5432"}
-: ${POSTGRES_USER:="airflow"}
-: ${POSTGRES_PASSWORD:="airflow"}
-: ${POSTGRES_DB:="airflow"}
+: ${POSTGRES_USER:="cdeAirflowMaster"}
+: ${POSTGRES_PASSWORD:="nottherealpassword"}
+: ${POSTGRES_DB:="cdeAirflow"}
```
Comment (stevelevine) [inline script/entrypoint.sh:15]: Is the RDS instance creation scripted?

Comment (nkijak) [reply script/entrypoint.sh:15]: no

File: docker-compose-LocalExecutor.yml (hunk 1 above)
Comment (nkijak) [reply docker-compose-LocalExecutor.yml:18]: its some signing key they use for communication between instances.  This compose file isn't being used. ECS uses its own thing.

File: script/entrypoint.sh (hunk 2 above)
Comment (stevelevine) [reply script/entrypoint.sh:15]: can you add story to do this?  we will need an easy way to spin up/down RDS instances.  We can work on it later.

**Pull request merged**